### PR TITLE
Unit tests & bug fixes

### PR DESCRIPTION
**Pull request by fmavb**
Created the pull request to test Travis CI for now

### CODE REVIEW

File: packages/core-hello/core-hello.js [hunk 1]
```diff
@@ -87,7 +87,7 @@ class CoreHello extends HTMLElement {
     // set new string in component
     let toSplit = this.innerText.split(" ");
     let name = toSplit[toSplit.length - 1];
-    console.log(name);
+    // console.log(name);
```
Comment (ShaneNielsen1234) [inline packages/core-hello/core-hello.js]: dead code

File: packages/core-hello/tests/core-hello.stories.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+ * 1. build the element with document.createElement
+ *
+ * Both are shown below.
+ */
+// TODO(Nate): Issue with multi word inner html only displaying last part
+storiesOf('core-hello', module)
+  .add('Default', () => '<core-hello lang="en">Peter</core-hello>')
+
+  //TODO(Nate) Currently broken with current implementation.
+  //.add('with Rainbow', () => '<core-hello lang="en">Peter</core-hello>')
```
Comment (ShaneNielsen1234) [inline packages/core-hello/tests/core-hello.stories.js]: Is there a ticket for this bug?

File: packages/core-hello/tests/core-hello_test.js [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+//----------Unit Tests ------------------
+describe('core-hello Unit Tests', () => {
+  it('Non Spec Language (Default) Inner Text', async () => {
+    // tests the english
+    let hwHTML = `<core-hello id='hw1' lang='en'>Shaya!</core-hello>`;
+    document.body.insertAdjacentHTML('afterbegin',hwHTML);
+    let hwEl = document.getElementById('hw1');
+    let hwText = hwEl.innerHTML;
+    //console.log("Component text: ", hwText);
```
Comment (ShaneNielsen1234) [inline packages/core-hello/tests/core-hello_test.js]: dead code. Sweep for dead code,

File: packages/core-hello/tests/core-hello.stories.js (hunk 2 above)
Comment (nonguyen1) [reply packages/core-hello/tests/core-hello.stories.js]: Just wrote one:)

**Pull request merged**